src/main/core/Manager/CurlManager.php

```diff
@@ -34,6 +34,7 @@ public function exec($url, $payload = null, $type = 'GET', $options = [], $autoC
         }
 
         curl_setopt($ch, CURLOPT_SSL_VERIFYPEER, false);
```

Comment:
This line seems wrong. I think I may have pushed this by mistake and is only required to make my local instance work.
Do you have any opinion on this @chalasr ?

Reply:
Unless we need to hit some https endpoint that don’t have a valid certificate, it should be removed yes